.github/workflows/ci.yml

```diff
@@ -51,7 +51,6 @@ jobs:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
       - run: julia -e 'using InteractiveUtils; versioninfo(verbose=true)'
-      - uses: julia-actions/cache@v1
```

Comment:
?

Reply:
But there seem to be some problems with deleting these caches, so should we give it a try and set the permissions?

Reply:
Sure, give it a try!